csrc/quantization/gptq_marlin/awq_marlin_repack.cu

```diff
@@ -122,7 +103,7 @@ __global__ void awq_marlin_repack_kernel(
     }
 
     uint32_t vals[8];
-  #pragma unroll
+#pragma unroll
```

Comment:
do you know why the whitespace is changing here?

Reply:
it appears to be clang-format not indenting any more because it is now longer in a:
```
#if defined(__CUDA_ARCH__) && __CUDA_ARCH__ < 800
... dummy code
#else
... was indenting here before
#endif
```
block